src/app/loadout/loadout-edit/LoadoutEdit.tsx

```diff
@@ -224,7 +224,7 @@ export function LoadoutEditSubclassSection({
       title={t('Bucket.Class')}
       onClear={handleClearSubclass}
       onRandomize={handleRandomizeSubclass}
-      onFillFromEquipped={handleFillSubclassFromEquipped}
+      onSyncFromEquipped={handleSyncSubclassFromEquipped}
```

Comment:
This renames the "Fill in from equipped" button in the Subclass section to "Sync from equipped" which more accurately describes its function -- it'll replace whatever is there.